feature-libs/order/styles/components/order-overview/_order-overview.scss

```diff
@@ -106,6 +108,13 @@
     flex-grow: 1;
     margin-bottom: 0;
 
+    @include forVersion(6.5) {
```

Comment:
I'm wondering if this also shouldn't be 6.6 version. Just wanted to confirm :)

Reply:
true, moved to 6.6